test/README.md

```diff
@@ -49,29 +49,36 @@ environment.
    myusername ALL=(ALL) NOPASSWD: /usr/bin/podman
    ```
 
-1. Install the `drenv` package in a virtual environment:
+1. Create a `ramen` virtual environment
 
-   Run this once in the root of the source tree:
+   Run this once:
 
    ```
-   python3 -m venv ~/.venv/ramen
-   source ~/.venv/ramen/bin/activate
-   pip install --upgrade pip
-   pip install -e ./test
+   make -C test venv
    ```
 
-   You can create the virtual environment anywhere, but keeping the
-   environment outside of the source tree is good practice.
+   This creates a python virtual environment in `~/.venv/ramen`, and
+   install the `drenv` tool using develop mode, so changes in `drenv`
+   source are available immediately without reinstalling the tool.
 
-   This installs a link file in the virtual environment:
+### Testing that drenv is healthy
 
-   ```
-   $ cat /home/nsoffer/.venv/ramen/lib/python3.10/site-packages/drenv.egg-link
-   /home/nsoffer/src/ramen/test
-   ```
+Run these steps to make sure `drenv` works in your setup:
+
+```
+source ~/.venv/ramen/bin/activate
+cd test
+drenv start example.yaml
```

Comment:
This step seems redundant given next section does the same thing.

Reply:
Sounds good. Maybe remove same content in next section then so it is not repeated.

Reply:
I don't see an issue in repeating the commands. User can copy and paste the commands
to a shell to run them.

Reply:
Should be resolved in current version.